docs/install/any-kubernetes-cluster.md

```diff
@@ -730,12 +730,24 @@ kubectl label namespace default eventing.knative.dev/injection=enabled
 
 {{< feature-state version="v0.2" state="alpha" >}}
 
-The following command installs the Github Source:
+The following command installs the single-tenant Github source:
 
 ```bash
 kubectl apply --filename {{< artifact repo="eventing-contrib" file="github.yaml" >}}
 ```
 
+The single-tenant GitHub source creates one Knative service per GitHub source.
+
+The following command installs the multi-tenant GitHub source:
+
+```bash
+kubectl apply --filename {{< artifact repo="eventing-contrib" file="mt-github.yaml" >}}
+```
+
+The multi-tenant GitHub source creates only one Knative service handling all
+GitHub sources in the cluster. This source does not support logging and tracing
+configuration yet.
```

Comment:
```suggestion
configurations yet.
```

Either "This source does not support logging and tracing configurations yet" or "This source does not support logging or tracing configuration yet" - Not sure which one is technically correct - if both require a configuration then it's prob and + configurations, if you are saying it doesn't support logging or a tracing configuration then maybe use or.

Reply:
fixed.